elasticdl/docker/Dockerfile.dev

```diff
@@ -37,23 +57,3 @@ RUN cd /tmp/elasticdl && make -f elasticdl/Makefile && \
     export GO111MODULE=on && \
     go mod init elasticdl.org/elasticdl && \
     go install ./... && cd / && rm -rf /tmp/elasticdl
-
-# Copy the data generation package to /var and run them from there.
```

Comment:
In my local test environment, I find that I have to build the elasticdl:dev if I modify Go code.
Just move commands not related with Go up, since it will be cached. This will reduce rebuild time.

Reply:
Better to put this change is a separate PR.
Looks fine to me otherwise.